dbms/src/Flash/Statistics/TableScanImpl.cpp

```diff
@@ -13,25 +13,35 @@
 // limitations under the License.
 
 #include <DataStreams/TiRemoteBlockInputStream.h>
+#include <Flash/Statistics/ConnectionProfileInfo.h>
 #include <Flash/Statistics/TableScanImpl.h>
 #include <Interpreters/Join.h>
 #include <Storages/DeltaMerge/ScanContext.h>
 
 namespace DB
 {
-String TableScanDetail::toJson() const
+String TableScanTimeDetail::toJson() const
 {
     auto max_cost_ms = max_stream_cost_ns < 0 ? 0 : max_stream_cost_ns / 1'000'000.0;
     auto min_cost_ms = min_stream_cost_ns < 0 ? 0 : min_stream_cost_ns / 1'000'000.0;
+    return fmt::format(R"("max":{},"min":{})", max_cost_ms, min_cost_ms);
+}
+String LocalTableScanDetail::toJson() const
+{
+    return fmt::format(R"({{"is_local":false,"bytes":{},{}}})", bytes, time_detail.toJson());
```

Comment:
why is_local is false for `LocalTableScanDetail`?

Reply:
Good catch, will change it